modules/installation-configuration-parameters.adoc

```diff
@@ -426,6 +422,13 @@ The value can also be the name of an existing Glance image, for example `my-rhco
    }
 }
 ----
+|`platform.openstack.ingressFloatingIP`
+|An existing floating IP address to associate with the Ingress port. To use this property, `platform.openstack.externalNetwork` must also be defined.
+|An IP address, for example `128.0.0.1`.
```

Comment:
Is it expected that the example IP addresses for `platform.openstack.ingressFloatingIP` and `platform.openstack.lbFloatingIP` are the same?

Reply:
It's not unexpected. It's to show appropriate values above all else. 

If you have a different address in mind, I'm happy to plug that in, though. ;)